core/src/contracts/stablex_contract.rs

```diff
@@ -330,6 +337,7 @@ impl StableXContract for StableXContractImpl {
     fn send_noop_transaction<'a>(
         &'a self,
         gas_price: U256,
+        nonce: U256,
```

Comment:
Can this method now be called `cancel_transaction`?

Reply:
I'm fine either way but this came up already in https://github.com/gnosis/dex-services/pull/1089#discussion_r450740145 .
Imo send_noop_transaction says exactly what this does since it still works even when there is no transaction to cancel.

Reply:
I'm on the fence about this one. If the signature was `fn(tx: ethcontract::web3::types::Transaction)` then I would definitely prefer the `cancel_transaction` name, but as is it isn't *really* cancelling a transaction, but sending a no-op transaction with the specified nonce and gas price. That being said, this method is exclusively used for cancelling transactions, so I also think it might make the name better describe what is its intended use (instead of what it does).

I'm pretty indifferent and am happy with both.

Reply:
As we now have changed the other method also to be called exactly what it does and not what it's intended for, I'm fine to leaving this here too. 
Just felt that stating the intention might help to understand what we are trying to do when we use it in another part of the codebase.